### DISCUSSION TITLE
When do you plan to release bud --cache-from / --cache-to functionality?

### DISCUSSION THREAD

**Discussion by ioboi** [containers/buildah#4161] [Q&A] [answered]
Just a quick question about your release roadmap after the merge of https://github.com/containers/buildah/pull/4118.

Thanks to everybody for your great work!

**Comment by rhatdan** [comment]
Sorry to say this missed the cut for v1.26.3 which should be released next week in distros.
The next release will probably be in a month or two.

**Reply by flouthoc** [reply]
@rhatdan From discussions here it seemed that `1.26.3` was a minor release which only included fixes and dependencies bump from upstream https://github.com/containers/buildah/pull/4152

**Reply by rhatdan** [reply]
Right, we should be cutting a 1.27 release?

**Reply by lukasmrtvy** [reply]
Will these PRs make it to the next release? And when? Thanks
https://github.com/containers/buildah/pull/4064
https://github.com/containers/buildah/pull/4071

**Reply by flouthoc** [reply]
@lukasmrtvy Yes, afaics these should be included in `v1.27.0` since they are feature change.

**Comment by flouthoc** [comment, marked as answer]
Buildah tag `v1.27.0` is here: https://github.com/containers/buildah/releases/tag/v1.27.0 which should contain requested features.

**Reply by ioboi** [reply]
Great, thanks to all involved :+1:

**Comment by ioboi** [comment]
Would it be possible to deploy the corresponding container image at: https://quay.io/repository/containers/buildah?tab=tags ?

**Reply by flouthoc** [reply]
Afaik it used to automatically create a tag but I'll tag @cevich for confirmation. I think branch https://github.com/containers/buildah/tree/release-1.27 needs some tweaking, is that right @cevich ?

**Reply by cevich** [reply]
Not in this case, for several complex reasons, the image is built from a `Containerfile` on `main`, but uses the released Fedora packages, so the Bohdi process has to go through first (see below).

**Comment by TomSweeneyRedHat** [comment]
From what I can parse, this fix made it into Buildah v1.27.0.  From what I see, that just became stable in the Fedora Project (https://bodhi.fedoraproject.org/updates/FEDORA-2022-936844c917).  Given that, our CI should pick up on that when it next runs the container image builds, and you should see the 1.27 tag in the quay.io/buildah/stable repo within 24 hours.  If not, we'll need @cevich on the case.

**Comment by cevich** [comment]
Wow! @TomSweeneyRedHat and @flouthoc are BOTH on the ball: If it breaks, call @cevich :grinning: 

Joking aside, I know it's a somewhat complex build process, so I'm happy to take a look if it breaks in some way.

**Reply by ioboi** [reply]
Hi @cevich 
There is still no 1.27 tag available at quay.io/buildah/stable repo.

**Reply by cevich** [reply]
Hrmmm, oh! https://bodhi.fedoraproject.org/updates/FEDORA-2022-936844c917 is for Fedora 37 which isn't released yet.  I don't know how to use this tool, but clicking around I find [a "build" for F36](https://koji.fedoraproject.org/koji/buildinfo?buildID=2043377) but don't know how to see it's updates status page :disappointed:  @TomSweeneyRedHat do you know how to find this?

**Comment by TomSweeneyRedHat** [comment]
I'm going to have to call in our Fedora ringer.  I'm only seeing Buildah v1.27 being built of Fedora 37 and not on Fedora 36.  @lsm5 do you know what's going on there?

**Reply by lsm5** [reply]
submitted a bodhi update just now https://bodhi.fedoraproject.org/updates/FEDORA-2022-fbbb3098ca . Give it karma so it can land in the stable repo soon.

**Comment by cevich** [comment]
Thanks @lsm5 once it lands in stable, the container image should be built & pushed 24hours later.

**Reply by ioboi** [reply]
Has landed https://quay.io/repository/containers/buildah?tab=history :tada:

**Comment by cevich** [comment]
Great, thanks for the confirmation.
